nbqa/__main__.py

```diff
@@ -38,7 +39,7 @@
     r"_build|buck-out|build|dist|venv"
     r")/"
 )
-
+TOKEN = secrets.token_hex(3)
```

Comment:
Minor: Instead of `secrets.token_hex` should we go with `hash(notebook_file_path)` so that its reproducible in  case of bug reports?

Reply:
good idea, thanks!